doc/PortingGuide.md

```diff
@@ -2,7 +2,7 @@ Requirements to Port Your Project to *cupla*
 ============================================
 
 - your build system must be `CMake`
```

Comment:
Unrelated to this PR, but I think this is not required anymore.

Reply:
Yes should be fixed separate